### DISCUSSION TITLE
Bump the Android SDK?

### DISCUSSION THREAD

**Discussion by bleez99** [NightscoutFoundation/xDrip#3114] [General]
Today, Android 14 was released and upon xDrip startup, Android complained about the application being built for an older version of Android.

Not an emergency, I OK'd the warning and everything is working fine. Pixel 7 Pro.

**Comment by Navid200** [comment]
![Screenshot_20231004-222452](https://github.com/NightscoutFoundation/xDrip/assets/51497406/c1d07ca0-42e5-40c3-9e23-6632b75edecc)

**Comment by corneliusroemer** [comment]
There are a number of Android 14 related discussions that could potentially be solved by bumping the SDK. I've made a collection issue here to track them all:
- https://github.com/NightscoutFoundation/xDrip/discussions/3141

**Reply by Navid200** [reply]
What number of ...?

Android 14 just was released recently, less than a month ago and I have already stated that this will be addressed.  We are volunteers.  It will be done.

WearOS needs someone who is willing to work to address the issues and feature requests.  Everyone is welcome to do that.

We are all volunteers here.  And it's not like we have no idea what needs to be addressed.  But, we need developers.  
Everyone has an idea.  We don't have a shortage of that.
We have a shortage of developers.

**Reply by corneliusroemer** [reply]
Thanks for your work - I'm not critcizing you at all. I was just reviewing issues and seeing whether people had come up with workarounds and felt it would be good to collect pointers to all Android 14 / wearos 4 issues. Feel free to ignore! I'll see whether I can have a look at the code and make sense of it - haven't done any Android/Java/Kotlin development so not sure how feasible.

**Reply by corneliusroemer** [reply]
I'm not saying all issues would be resolved by bumping the SDK, it's just useful to see that there might be some issues with Android 14 in contrast to what OP said ("everything is working fine")

**Reply by Navid200** [reply]
For your info, when Android 10 came out, the xDrip Dexcom collector had to be modified.
Later, when an Android 11 patch came out, the xDrip Dexcom collector had to be modified again.
Every time, there were xDrip users who did not have working systems until xDrip was fixed.
None of that has happened with Android 14.   Therefore, everything is indeed working fine with Android 14.

If you have an issue using xDrip to get readings from Dexcom, Libre or anything else for that matter, since upgrading to Android 14, please report it.
At this point, there has been none.

WearOS is something else and I have already explained that.
